Frontend/ui-driver/src/Screens/RideSummaryScreen/ComponentConfig.purs

```diff
@@ -0,0 +1,354 @@
+module Screens.RideSummaryScreen.ComponentConfig where
+
+import Prelude
+
+import Components.SourceToDestination as SourceToDestination
+import Components.RideSummaryCard as RideSummaryCard
+import Screens.RideSummaryScreen.ScreenData
+import Data.Maybe
+import PrestoDOM
+import Helpers.Utils (fetchImage, FetchImageFrom(..))
+import Styles.Colors as Color
+import Font.Size as FontSize
+import Font.Style as FontStyle
+import Services.API 
+import Common.Types.App as CTA
+import Components.SourceToDestination
+import Resource.Constants
+import Helpers.Utils
+import Data.String (split, Pattern(..))
+import Data.Array (mapWithIndex , (!!), length, null)
+import Components.DropDownCard as DropDownCard
+import Effect (Effect)
+import Language.Strings (getString)
+import Language.Types as LType
+import Data.Int as INT
+import Components.PopUpModal as PopUpModal
+import Screens.Types as ST
+import Language.Strings (getString)
+import Language.Types (STR(..)) as LT
+import Engineering.Helpers.Commons as EHC
+import PrestoDOM.Types.DomAttributes (Corners(..)) as PTD
+import Services.API (ServiceTierType(..))
+import Debug
+
+
+dummyLocation :: Location
+dummyLocation = Location {
```

Comment:
This was present other places also. Move to a common place. Make Just "" to Nothing